lxd/container_lxc.go

```diff
@@ -1811,6 +1811,13 @@ func (c *containerLXC) initLXC(config bool) error {
 		return err
 	}
 
+	if !c.isCurrentlyPrivileged() && !c.state.OS.RunningInUserNS && lxc.HasApiExtension("seccomp_notify") && c.DaemonState().OS.SeccompListener {
```

Comment:
This is done at startup, so IsPrivileged() would actually be easier to read here, isCurrentlyPrivileged is only really relevant for running containers.